### PR TITLE
 added changed information to text element so we can mark it as inserted or deleted

### PR DESCRIPTION
**Pull request by moromete**
If you want to enable track changes on added text you can mark it as INSERTED or DELETED by a specific user at a given time:

```
$text = $section->addText('Hello World!');
$text->setChanged(\PhpOffice\PhpWord\Element\ChangedElement::TYPE_INSERTED, 'Fred', time());
```

### CODE REVIEW

**Comment by moromete** [comment]
Can you please tell me why this was removed from  v0.13.0 ? Is there any problem we can solve ...

**Comment by troosan** [comment]
I guess at the time they did not have the time to include in 0.13
I did not check the code yet, but this should be merged with the PhpOffice\PhpWord\Element\TrackChange type, which is currently only used to add Comments.

see possible extensions here http://datypic.com/sc/ooxml/t-w_CT_TrackChange.html

**Comment by moromete** [comment]
OK I updated the ChangedElement Type to extend from TrackChange and also added the reader from docx and odt to detect tracking changes inside the document.

File: src/PhpWord/Reader/Word2007/AbstractPart.php [hunk 1]
```diff
@@ -98,8 +98,7 @@ public function setRels($value)
      *
      * @todo Get font style for preserve text
      */
-    protected function readParagraph(XMLReader $xmlReader, \DOMElement $domNode, $parent, $docPart = 'document')
-    {
+    protected function readParagraph(XMLReader $xmlReader, \DOMElement $domNode, $parent, $docPart = 'document') {
```
Comment (troosan) [inline src/PhpWord/Reader/Word2007/AbstractPart.php]: put the `{` on the next line to make the build succeed.

**Comment by troosan** [comment]
It was not enough:
https://travis-ci.org/PHPOffice/PHPWord/jobs/328988900

**Comment by troosan** [comment]
@moromete since I cannot commit on your master branch I pulled your branch into another one and modified the code a bit. I also added unit tests to cover the new code.

**Pull request closed without merging**